### PR TITLE
User name set as 'Group Name' fixed

### PR DESCRIPTION
**Pull request by joshsadam**
## Description of changes

Updated column name for `User` in the Admin User Groups table

## Related issue

N/A

## Checklist

Things for the developer to confirm they've done before the PR should be accepted:

*   [x] CHANGELOG.md (and UPGRADING.md if necessary) updated with information for new change.
*   [ ] ~Tests added (or description of how to test) for any new features.~
*   [ ] ~User documentation updated for UI or technical changes.~

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,7 @@
 
 ## [22.09.6] - 2022/12/21
 * [UI]: Fixed bug on NCBI Export page preventing the export from occuring. See [PR 1439](https://github.com/phac-nml/irida/pull/1439)
+* [UI]: Fixed bug where the `User` column was named `User Group` on the admin User Groups page. [See PR 1450](https://github.com/phac-nml/irida/pull/1450)
```
Comment (ericenns) [inline CHANGELOG.md:13]: This should be in the section above

**Review by ericenns — CHANGES_REQUESTED** [review]
Changelog comment is in the wrong place, otherwise good.

File: CHANGELOG.md (hunk 1 above)
Comment (joshsadam) [reply CHANGELOG.md]: You are correct, Fixed

**Review by ericenns — APPROVED** [review]
👍

**Pull request merged**